core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -242,7 +249,11 @@ export class QuestionSuggestionReviewModalComponent
     this.showQuestion = false;
     this.skippedContributionIds.push(this.currentSuggestionId);
 
-    this.currentSuggestionId = this.remainingContributionIdStack.pop();
+    let CurrentSuggestionId = this.remainingContributionIdStack.pop();
```

Comment:
```suggestion
    const currentSuggestionId = this.remainingContributionIdStack.pop();
```

Reply:
Done

Reply:
Not done? Note that the variable should be uncapitalized as well.

Reply:
Done.